spec/jsSpecs/ares-log.spec.js

```diff
@@ -0,0 +1,385 @@
+/*
+ * Copyright (c) 2021 LG Electronics Inc.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/* eslint-disable no-useless-escape */
+
+const exec = require('child_process').exec,
+    path = require('path'),
+    fs = require('fs'),
+    common = require('./common-spec');
+
+const aresCmd = 'ares-log',
+    savedlogPath = path.join(__dirname, "..", "tempFiles", "savedlog.log"),
+    exp = /\w+ \d+ \d\d:\d\d:\d\d [\w|\d]+ [\w|\d|\.|\-]+\[\d+]:/g;
```

Comment:
Could you please what is exp as comment or change value name to make more clear.